mem/file.go

```diff
@@ -81,6 +83,14 @@ func SetMode(f *FileData, mode os.FileMode) {
 	f.mode = mode
 }
 
+func SetUID(f *FileData, uid int) {
+	f.uid = uid
```

Comment:
This and the SetGID function will need locks, like they were recently added to master to not have races on the FileData struct.